### PR TITLE
Added `command_test.py`.

### LINKED ISSUE #27: Write integration tests for commands

### PR DESCRIPTION
**Pull request by konstantin-ogulchansky**
Closes #27.

### CODE REVIEW

File: tests/integration/command_test.py [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+    @two.command
+    def x():
+        return True
+
+    @two.command
+    def y():
+        return False
+
+    # Act.
+    cx_a = execute(with_='one a')
```
Comment (JoshuaLight) [inline tests/integration/command_test.py:59]: I feel like it's probably would be better to reduce number of commands and checks in this test. In an ideal case, _Assert_ and _Act_ sections should occupy only one line each.

Maybe it's possible to rewrite the execution to only `cx = execute(with_='one a')` to check whether a sub-sub-command is called correctly?

File: tests/integration/command_test.py [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+    @main.command
+    def first(x: Arg[int]):
+        pass
+
+    @first.command
+    def second(y: Opt[int]):
+        pass
+
+    @second.command
+    def third(z: Flag):
```
Comment (JoshuaLight) [inline tests/integration/command_test.py:131]: I think it's enough to test only 2 levels of deepness. If it works for `second`, then it'll probably work for any level.

One exception I could think of is when we're trying to test upper bound for number of sub-commands a CLI could have.

File: tests/integration/command_test.py [hunk 3]
```diff
@@ -0,0 +1,241 @@
...
+    cx_y = execute(with_='two y')
+
+    # Assert.
+    assert cx_a is True
+    assert cx_b is False
+    assert cx_x is True
+    assert cx_y is False
+
+
+def test_invalid_command_name():
```
Comment (JoshuaLight) [inline tests/integration/command_test.py:71]: I'm not quite sure what does the `invalid_command_name` mean? Is it just a name of the command, or how it was specified in the command line?

File: tests/integration/command_test.py (hunk 1 above)
Comment (konstantin-ogulchansky) [reply tests/integration/command_test.py:59]: Why?

File: tests/integration/command_test.py (hunk 2 above)
Comment (konstantin-ogulchansky) [reply tests/integration/command_test.py:131]: Agree.

File: tests/integration/command_test.py (hunk 3 above)
Comment (JoshuaLight) [reply tests/integration/command_test.py:71]: It's possible to reduce this test case to the following:
```py
# Arrange.
def test_one_subcommand_specified_incorrectly():
    @cli
    def main():
        pass

    @main.command
    def one():
        pass

    @one.command
    def two():
        pass

# Act.
ex = execute('one tw_')

# Assert.
...
```

And we also could have a mirror test case that checks happy path:
```py
# Arrange.
def test_one_subcommand():
    @cli
    def main():
        pass

    @main.command
    def one():
        pass

    @one.command
    def two():
        return True

# Act.
cx = execute('one two')

# Assert.
assert cx == True
```

Comment (konstantin-ogulchansky) [reply tests/integration/command_test.py:71]: In this case, the name of an inexistent command is meant. Will rename the test to make it more clear.

**Pull request merged**